SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Uyuni follows a rolling release model, so it is always the next upcoming version that will contain bugfixes for the current one as well as any new features. As there will be no maintenance of specific versions, users will generally need to update to the most recent version in order to receive the latest security updates. For very critical bugfixes and vulnerabilities though in terms of impact and risk, we might decide to release a patch, but this will always be a case by case decision.
```

Comment:
> For very critical bugfixes and vulnerabilities though in terms of impact and risk, we might decide to release a patch, but this will always be a case by case decision.

We need to talk about this on Monday. I am not exactly how to explain it, but don't think this is the right wording according to what we discussed today.

Reply:
Changed the text indicating that there will be patches for critical bugs only, while all others will usually be fixed with the next rolling release.